src/python/WMCore/MicroService/MSPileup/MSPileupTasks.py

```diff
@@ -62,6 +62,31 @@ def pileupSizeTask(self):
             msg = f"MSPileup pileup size task failed with error {exp}"
             self.logger.exception(msg)
 
+    def cleanupTask(self, timeThreshold=15):
+        """
+        Execute cleanup task according to the following logic:
+
+        1. Fetch documents from backend database for the following conditions
+           - active=False, and
+           - empty ruleIds list; and
+           - empty currentRSEs; and
+           document has been deactivated for a while (deactivatedOn=XXX),
+        2. For those documents which are fetched make delete call to backend database
+
+        :param timeThreshold: time threshold in days which will determine document clean-up readiness
+        """
+        spec = {'active': False}
+        docs = self.mgr.getPileup(spec)
+        deleteDocs = 0
+        thr = timeThreshold * 24 * 60 * 60  # convert to second
```

Comment:
I know you can give `thr` a more meaningful name ;)

Reply:
done